client/services/PlanningStoreService.js

```diff
@@ -0,0 +1,102 @@
+import { isNil } from 'lodash'
+import { createStore } from '../utils'
+import * as actions from '../actions'
+
+PlanningStoreService.$inject = [
+    '$rootScope',
+    'api',
+    'config',
+    '$location',
+    '$timeout',
+    'vocabularies',
+    'superdesk',
+    'upload',
+    'notify',
+    'privileges',
+    'notifyConnectionService',
+    'userList',
+    'desks',
+    'metadata',
+    'session',
+    'deployConfig',
+    'gettextCatalog',
+]
+export function PlanningStoreService(
+    $rootScope,
+    api,
+    config,
+    $location,
+    $timeout,
+    vocabularies,
+    superdesk,
+    upload,
+    notify,
+    privileges,
+    notifyConnectionService,
+    userList,
+    desks,
+    metadata,
+    session,
+    deployConfig,
+    gettextCatalog
+) {
+    let self = this
+
+    self.store = null
```

Comment:
I'm not sure if all this loading should be done via actions dispatching, imo would be better to fetch it all and only create the store after. Ideally this would be fetched via [route.resolve](https://docs.angularjs.org/api/ngRoute/provider/$routeProvider). Because before it's all loaded there is no point really to render anything.